src/app/series/directives/series-podcast.component.html

```diff
@@ -66,12 +49,51 @@
       </div>
     </publish-fancy-field>
 
+    <publish-fancy-field label="Podcast Audio" [select]="audioVersionOptions" [model]="distribution" name="versionTemplateUrl" required>
+      <div class="fancy-hint">Select which audio template should be used with your podcast.</div>
+    </publish-fancy-field>
+
     <publish-fancy-field label="Homepage Link" required textinput [model]="podcast" name="link">
       <div class="fancy-hint">A link to the homepage for your podcast.</div>
     </publish-fancy-field>
 
-    <publish-fancy-field label="Podcast Audio" [select]="audioVersionOptions" [model]="distribution" name="versionTemplateUrl" required>
-      <div class="fancy-hint">Select which audio template should be used with your podcast.</div>
+    <publish-fancy-field label="Author Information">
+      <div class="fancy-hint">Set the author name and email to be associated with this podcast.</div>
+      <div class="span-fields">
+        <publish-fancy-field label="Name" textinput [model]="podcast" name="authorName" small=1>
+        </publish-fancy-field>
+        <publish-fancy-field label="Email" textinput [model]="podcast" name="authorEmail" small=1>
+        </publish-fancy-field>
+      </div>
+    </publish-fancy-field>
+
+    <publish-fancy-field label="PRX Feed" class="prx-feed-url" *ngIf="!distribution.isNew">
+      <div class="fancy-hint" *ngIf="podcast?.hasPublicFeed">
+        The private URL for your PRX podcast feed, providing the content for <a href="{{podcast.publicFeedUrl}}">your public feed</a>.
+      </div>
+      <div class="fancy-hint" *ngIf="!podcast?.hasPublicFeed">
+        The URL for your PRX podcast feed. It's encouraged for podcasters to use a  public proxy like feedburner (potentially with a custom domain)
+        for sharing with listeners. If you already have a public URL for your podcast feed, enter it in the next field, or feel free to
+        <a href="https://support.google.com/feedburner/answer/78475?hl=en" title="Feedburner Help">create one.</a>
+      </div>
+      <input type="text" readonly [ngModel]="podcast?.publishedUrl" name="publishedUrl"/>
```

Comment:
Seems like we should have width: 100% on this input instead of the same width that applies to urls with the copy/open link buttons.